openshift_cli_installer/libs/managed_clusters/osd_clusters.py

```diff
@@ -72,12 +73,12 @@ def osd_create_cluster(cluster_data, must_gather_output_dir=None):
             f"Failed to run cluster create for cluster {cluster_name}\n{ex}",
             fg=ERROR_LOG_COLOR,
         )
+        set_cluster_auth(cluster_data=cluster_data, cluster_object=cluster_object)
```

Comment:
Now `cluster_data` contain `cluster_object`, no need to pass it to `set_cluster_auth`

Reply:
yes we do; we need the file

Reply:
Once you set it in `openshift_cli_installer.utils.cli_utils.prepare_clusters` you can take it from `cluster_data`

Reply:
ahhh